### PR TITLE
Upload project and locations

### PR DESCRIPTION
**Pull request by dimasciput**
Progress : 
cadasta-project-creation

### CODE REVIEW

File: cadasta/gui/tools/wizard/step_login01.py [hunk 1]
```diff
@@ -50,6 +60,44 @@ def set_widgets(self):
...
+            self.clear_information
+        )
+        self.url_input.setText(get_url_instance())
+
+        # If login information exists
+        if get_authtoken():
+            self.clear_button.setEnabled(True)
+            self.username_input.setText(get_setting('username'))
+            self.token_status.setText(
+                tr(
```
Comment (ismailsunni) [inline cadasta/gui/tools/wizard/step_login01.py]: I think it's fine to put it like this:
```
tr('Auth token is saved.')
```
in one line

File: cadasta/gui/tools/wizard/step_project_creation01.py [hunk 2]
```diff
@@ -98,18 +98,15 @@ def validate_step(self):
 
         if self.project_url() and \
                 not is_valid_url(self.project_url()):
-            error_message += tr(
+            error_message = tr(
```
Comment (ismailsunni) [inline cadasta/gui/tools/wizard/step_project_creation01.py]: This one also, make it one line for less verbose code

File: cadasta/gui/tools/wizard/step_project_creation01.py [hunk 3]
```diff
@@ -160,7 +157,7 @@ def all_data(self):
         data['project_url'] = self.project_url()
         data['contact'] = {
             'name': self.contact_name_text.displayText(),
-            'phone': self.contact_phone_text.displayText(),
+            'tel': self.contact_phone_text.displayText(),
```
Comment (ismailsunni) [inline cadasta/gui/tools/wizard/step_project_creation01.py:156]: Is there any specific reason to use `tel`? Why not full word, i.e. telephone?

**Review by ismailsunni — COMMENTED** [review]
There are some multiline that can be written as one line (I made a comment to one of them). Less line of code is better since it will be easier to read in the screen.

Please fix the travis also. LGTM

**Comment by ismailsunni** [comment]
Looks Great

File: cadasta/gui/tools/wizard/step_project_creation01.py (hunk 3 above)
Comment (dimasciput) [reply cadasta/gui/tools/wizard/step_project_creation01.py:156]: I'm following their api contract https://cadasta.github.io/api-docs/#create-a-new-project

**Pull request merged**